addons/mail/static/src/components/chat_window_header/chat_window_header.xml

```diff
@@ -9,10 +9,10 @@
                         <i class="fa fa-arrow-left"/>
                     </div>
                 </t>
-                <t t-if="chatWindow.thread and chatWindow.thread.model === 'mail.channel'">
+                <t t-if="chatWindowHeaderView.threadIconView'">
```

Comment:
typo: remove `'` (antepenultimate character on this line, without considering EOL)